### PR TITLE
Creating components for predictions game

### PR DESCRIPTION
**Pull request by Kev-Y-Huang**
Image of the binary choice, multiple choice, and interval choice type questions
image

### CODE REVIEW

File: web/src/components/predictions-components/multiple-category-choice.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import { Box, Label } from "theme-ui";
+import { Range, getTrackBackground } from 'react-range';
+
+const STEP = 0.1;
+
+class MultipleCategoryChoice extends Component {
```
Comment (mwjkwan) [inline web/src/components/predictions-components/multiple-category-choice.js:7]: Could you try writing this component as a function with [useState](https://reactjs.org/docs/hooks-state.html)?

File: web/src/components/predictions-components/multiple-category-choice.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import { Box, Label } from "theme-ui";
+import { Range, getTrackBackground } from 'react-range';
+
+const STEP = 0.1;
```
Comment (mwjkwan) [inline web/src/components/predictions-components/multiple-category-choice.js:5]: Should step also be a prop?

File: web/src/components/predictions-components/interval-choice.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    <form onSubmit={event => afterSubmission(event)}>
+      <Box mt={1} mx={3}>
+        <Range
+          draggableTrack
+          values={values}
+          step={STEP}
+          min={lower}
+          max={upper}
+          onChange={values => setValues(values)}
+          renderTrack={({ props, children }) => (
```
Comment (mwjkwan) [inline web/src/components/predictions-components/interval-choice.js:26]: I have a slight preference for defining Track and Thumb as separate components within this file... let me know what you think!

File: web/src/components/predictions-components/multiple-category-choice.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+                      style={{
+                        position: 'absolute',
+                        top: '-2rem',
+                        color: '#fff',
+                        fontWeight: 'bold',
+                        fontSize: '1rem',
+                        fontFamily: 'Arial,Helvetica Neue,Helvetica,sans-serif',
+                        padding: '0.3rem',
+                        borderRadius: '0.3rem',
+                        backgroundColor: '#2F2F2F'
```
Comment (mwjkwan) [inline web/src/components/predictions-components/multiple-category-choice.js:116]: Make sure to run `prettier --write "**/*.js"` on the file so it standardizes these quotes

**Review by mwjkwan — CHANGES_REQUESTED** [review]
The components look awesome! After a couple minor changes, I'm down to merge tonight and then we can coordinate with @matthewqu14 from there.

File: web/src/components/predictions-components/multiple-category-choice.js (hunk 1 above)
Comment (Kev-Y-Huang) [reply web/src/components/predictions-components/multiple-category-choice.js:7]: Yep, I can check that out

File: web/src/components/predictions-components/multiple-category-choice.js (hunk 2 above)
Comment (Kev-Y-Huang) [reply web/src/components/predictions-components/multiple-category-choice.js:5]: yeah, I can create a step prop

File: web/src/components/predictions-components/interval-choice.js (hunk 3 above)
Comment (Kev-Y-Huang) [reply web/src/components/predictions-components/interval-choice.js:26]: Thumb was implementable as a component but was having some trouble with Track and the ref prop

Comment (Kev-Y-Huang) [reply web/src/components/predictions-components/interval-choice.js:26]: actually, just figured out a fix

**Review by mwjkwan — APPROVED** [review]

**Pull request merged**